Dockerfile

```diff
@@ -1,30 +1,36 @@
+ARG PROJ_VERSION="6.1.0"
+
 FROM rust:1-slim-stretch as builder
+ARG PROJ_VERSION
+ENV PROJ_DEB "proj_${PROJ_VERSION}_amd64.deb"
+ENV DESTDIR "/build_proj"
+ENV GPG_KEY "C60D758F807A525534C5DFD57B639E3638A8112A"
+
 WORKDIR /usr/src/transit_model
-ARG DESTDIR="/build_proj"
-ARG PROJ_VERSION="6.1.0"
-RUN apt update && apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev libssl-dev clang \
-	&& wget https://github.com/OSGeo/proj.4/releases/download/$PROJ_VERSION/proj-$PROJ_VERSION.tar.gz \
-	&& tar -xzvf proj-$PROJ_VERSION.tar.gz \
-	&& cd proj-$PROJ_VERSION \
-	&& ./configure --prefix=/usr \
-	&& make \
-	&& make install \
-	&& cd .. && rm -rf proj-$PROJ_VERSION proj-$PROJ_VERSION.tar.gz \
-	&& apt-get purge -y wget build-essential sqlite3 libsqlite3-dev \
-	&& apt-get autoremove -y \
-	&& rm -rf /var/lib/apt/lists/* \
-	&& cp -r /build_proj/usr/include/* /usr/include \
-	&& cp -r /build_proj/usr/lib/* /usr/lib
+RUN apt update && apt install --yes apt-transport-https gnupg2
+RUN echo "deb [arch=amd64] https://kisiodigital.jfrog.io/kisiodigital/debian-local stretch main" > /etc/apt/sources.list.d/kisio-digital.list
+RUN gpg2 --receive-keys ${GPG_KEY}
```

Comment:
It's possible to get the key in a different way, like TravisCI is doing. Something like
```
curl -sSL "https://kisiodigital.jfrog.io/kisiodigital/api/gpg/key/public" | sudo -E apt-key add -
```
We might need to use `wget` instead (because `curl` is probably not present on a stretch distribution) but you get the idea.  I'll try to work on that.

Reply:
I just made the update so now, it's using `apt-key` to import the key directly from the Artifactory instance (and not from a GPG key server anymore).